OpenRA.Mods.Common/Traits/Upgrades/UpgradeManager.cs

```diff
@@ -125,6 +125,14 @@ public void RevokeUpgrade(Actor self, string upgrade, object source)
 			NotifyUpgradeLevelChanged(s.Traits, self, upgrade, -1);
 		}
 
+		//Returns true if the actor can work with the upgrade. Does not check the actual level of the upgrade.
```

Comment:
What does "can work with the upgrade" mean?
Also these comments should be `<summary>`.